### DISCUSSION TITLE
Assets folder?

### DISCUSSION THREAD

**Discussion by minor** [vercel/next.js#15490] [Help] [answered]
Hi!

I was exploring through this blog and I see that the blog posts themselves are referencing things from an assets folder (i.e. author photos, cover photos, etc). How do you access this? I've looked in the GitHub repository and even Vercel website with no luck.

Thanks in advance!

**Comment by timneutkens** [comment, marked as answer]
Is this what you're looking for? https://nextjs.org/docs/basic-features/static-file-serving

**Reply by minor** [reply]
Oh oops, yeah it is. Thanks!
